content/pages/om-oss.mdx

```diff
@@ -1,10 +1,41 @@
 ---
 title: Om oss
-description: fawekgnaejkgnewkøasg
+description: Informasjon om Queer Hangout
 slug: /om-oss
 menu: 1
 ---
 
 # Om oss
 
-Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+**Queer Hangout** er en møteplass for skeive voksne i Drammen. Vi er en frivillig organisasjon som møtes to ganger i måneden. Vi har også en egen facebookgruppe hvor folk kan legge ut egne aktiviteter, som bursdager, juleverksted, skogsturer eller frokost på 17.mai.  
+  
+Queer Hangout er ikke bare en vennegjeng, men også en gruppe aktivister som jobber mot ensomhet i det skeive miljøet. Blant annet er mange funksjonshemmede utelatt pga. manglende tilrettelegging. Vi har derfor bygget en egen rullestolrampe, ordnet stillerom og valgt tilrettelagte lokaler. Vi samarbeider med hvert medlem og relevante organisasjoner, sånn at vi får gjort ting riktig.  
+  
+![En frokost på 17.mai, innendørs i en leilighet. 2 personer ser på hverandre og ler, med et glass champagne i hånden. Første personen er i en fin dress, den andre er i samekofte. Det er salat og brødmat over bordet, med 17.mai pynt som små norske flagg](../images/17mai.jpeg)  
+![6 personer sitter på tepper på gresset ved drammenselva. Det er en piknikk, med mange fargefulle tepper, mat, drikke og en høytaler. Det er sol og godt sommervær, og nesten alle i bildet har på solbriller og en solhatt. De smiler og ser mot kameraet som tar dette bildet. Folk sitter nærme hverandre, noen med et glass i hånden. En rullestol står rett bak en person](../images/piknik.jpg)  
+
+  
+Queer Hangout er fylt med "fiksere", så vi har klart å gjøre mye med lite. Alt, fra denne websiden til rullestolrampen, er det noen i Queer Hangout som har laget på dugnad. Men det er mye vi ikke kan gjøre uten penger, og vi setter derfor stor pris på alle donasjoner! 
+  
+Her er våre små og større mål.  
+  
+**Sunn, hjemmelaget mat**  
+Skeive er mer utsatt for diskriminering i arbeidslivet, og har ofte dårligere råd enn resten av befolkningen. Vi tilbyr derfor et sunt gratis måltid på våre treff, som vi lager sammen som en sosial aktivitet. Dette er en av våre største utgifter. En donasjon på 150 kr. vil gi et måltid til alle på en Queer Hangout (gjennomsnittlig 20 personer). Man kan vippse oss på #818416, tusen takk!  
+  
+**Skeivt ball under pride**  
```

Comment:
Bør være undertittel, et nivå dypere enn tittelen over. Dette er viktig for accessibility.